lesson05.md

```diff
@@ -0,0 +1,397 @@
+<!-- .slide: id="lesson5" -->
+
+# Basic Frontend - Spring 2021
+
+Lesson 5, Tuesday, 2021-03-16
+
+---
+
+### Recap
+
+| Group | Operators | Example |
+| ----- | --------- | ------- |
+| Numerical Operators | `+ - * / **` | `5 + 4 * 3`<br/>`7 / 2 - 2`<br/>`"Hello" + " World"` |
+| Comparison Operators | `=== !== < > <= >=` | `temperature !== 25`<br/>`age >= 18` |
+| Logical Operators | &#124;&#124; `&& !` | `a && !b`<br />`x >= 5 && x < 15` |
+| Combined Operators | `+= -= *= /= ++ --` | `a *= 2`<br />`count++` |
+
+---
+
+### Homework solution
+
+```js
+let myAge = 42;
+let dogAge = 6;
+console.log("My dog is older:", dogAge * 7 > myAge);
+myAge++;
+dogAge++;
+console.log("My dog is older next year:", dogAge * 7 > myAge);
+```
+
+---
+
+```js
+let a = 42;
+let b = 12;
+
+console.log("Values of a and b are", a, b);
+
+/* YOUR CODE HERE. LEAVE CODE ABOVE AND BELOW! */
+let temp = a;
+a = b;
+b = temp;
+
+console.log("New values of a and b are", a, b);
+```
+
+---
+
+```js
+let orderedTea = true;
+let orderedCoffee = false;
+let orderedCake = false;
+let getsExtraCookie = orderedCake && (orderedTea || orderedCoffee);
+```
+
+---
+
+<!-- .slide: id="if" -->
+# Conditional Statements
+
+---
+
+### Conditional Statements
+
+Up until now, we wrote code line by line.
+
+But what if I want to run some code only if a certain condition is true or false? For example, only run code when the user is logged in?
+
+---
+
```

Comment:
Can we add an image here - https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/conditionals/cookie-choice-small.png